types/mjml/index.d.ts

```diff
@@ -1,46 +1,12 @@
-// Type definitions for mjml 4.0
+// Type definitions for mjml 4.7
 // Project: https://github.com/mjmlio/mjml, https://mjml.io
-// Definitions by: aahoughton <https://github.com/aahoughton>
-//                 marpstar   <https://github.com/marpstar>
+// Definitions by: aahoughton         <https://github.com/aahoughton>
+//                 marpstar           <https://github.com/marpstar>
 //                 eiskalteschatten   <https://github.com/eiskalteschatten>
-//                 emrah88    <https://github.com/emrah88>
+//                 emrah88            <https://github.com/emrah88>
+//                 IanEdington        <https://github.com/IanEdington>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-interface MJMLParsingOpts {
-    fonts?: { [key: string]: string; };
-    keepComments?: boolean;
-    beautify?: boolean;
-    minify?: boolean;
-    validationLevel?: 'strict' | 'soft' | 'skip';
-    filePath?: string;
-    minifyOptions?: MJMLMinifyOptions;
-}
+import mjml2html from 'mjml-core';
 
-interface MJMLParseError {
-    line: number;
-    message: string;
-    tagName: string;
-    formattedMessage: string;
-}
-
-interface MJMLParseResults {
-    html: string;
-    errors: MJMLParseError[];
-}
-
-interface MJMLJsonObject {
-    tagName: string;
-    attributes: object;
-    children?: MJMLJsonObject[];
-    content?: string;
-}
-
-interface MJMLMinifyOptions {
-    collapseWhitespace?: boolean;
-    minifyCSS?: boolean;
-    removeEmptyAttributes?: boolean;
-}
-
-declare function mjml2html(inp: string | MJMLJsonObject, opts?: MJMLParsingOpts): MJMLParseResults;
-
-export = mjml2html;
+export default mjml2html;
```

Comment:
reverting this based on https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default